docs/tasks/2d_face_keypoint.md

```diff
@@ -9,6 +9,7 @@ MMPose supported datasets:
 - [WFLW](#wflw-dataset) \[ [Homepage](https://wywu.github.io/projects/LAB/WFLW.html) \]
 - [AFLW](#aflw-dataset) \[ [Homepage](https://www.tugraz.at/institute/icg/research/team-bischof/lrs/downloads/aflw/) \]
 - [COFW](#cofw-dataset) \[ [Homepage](http://www.vision.caltech.edu/xpburgos/ICCV13/) \]
+- [COCO-WholeBody-Face](#coco-wholebody-face) \[ [Homepage](https://github.com/jin-s13/COCO-WholeBody/) \]
```

Comment:
face or facehand?

Reply:
COCO-WholeBody, COCO-WholeBody-Face, COCO-WholeBody-Hand  

we make it 3 different subsets